applications/nrf_desktop/configuration/nrf54l15pdk_nrf54l15_cpuapp/prj_keyboard.conf

```diff
@@ -0,0 +1,110 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+################################################################################
+# ZDebug: Keyboard version
+#
+# This ZDebug build type configuration acts as a keyboard.
+#
+# Application Configuration
+
+CONFIG_DESKTOP_INIT_LOG_HID_REPORT_EVENT=n
+CONFIG_DESKTOP_INIT_LOG_HID_REPORT_SENT_EVENT=n
+CONFIG_CAF_INIT_LOG_KEEP_ALIVE_EVENTS=n
+
+CONFIG_DESKTOP_ROLE_HID_PERIPHERAL=y
+CONFIG_DESKTOP_PERIPHERAL_TYPE_KEYBOARD=y
+CONFIG_DESKTOP_DEVICE_PID=0x52DD
+CONFIG_DESKTOP_HID_BOOT_INTERFACE_KEYBOARD=y
+CONFIG_DESKTOP_HID_STATE_HID_KEYMAP_DEF_PATH="hid_keymap_def_keyboard.h"
+CONFIG_DESKTOP_HID_STATE_HID_KEYBOARD_LEDS_DEF_PATH="hid_keyboard_leds_def_keyboard.h"
+CONFIG_DESKTOP_HID_REPORT_EXPIRATION=10000
+CONFIG_DESKTOP_HID_EVENT_QUEUE_SIZE=60
+
+CONFIG_CAF_BUTTONS=y
+CONFIG_CAF_BUTTONS_POLARITY_INVERSED=y
+CONFIG_CAF_BUTTONS_PM_KEEP_ALIVE=n
+
+CONFIG_CAF_CLICK_DETECTOR=y
+
+CONFIG_CAF_LEDS=y
+
+CONFIG_DESKTOP_BUTTONS_SIM_ENABLE=y
+CONFIG_DESKTOP_BUTTONS_SIM_INTERVAL=100
+CONFIG_DESKTOP_BUTTONS_SIM_LOOP_FOREVER=y
+CONFIG_DESKTOP_BUTTONS_SIM_TRIGGER_KEY_ID=0x01
+
+CONFIG_DESKTOP_BLE_PEER_CONTROL=y
+CONFIG_DESKTOP_BLE_PEER_CONTROL_BUTTON=0x0000
+CONFIG_DESKTOP_BLE_PEER_ERASE_ON_START=y
+
+CONFIG_DESKTOP_BLE_USE_DEFAULT_ID=y
+
+CONFIG_DESKTOP_BLE_SECURITY_FAIL_TIMEOUT_S=10
+CONFIG_DESKTOP_BLE_LOW_LATENCY_LOCK=y
+
+CONFIG_DESKTOP_CONFIG_CHANNEL_ENABLE=y
+# The nRF54L target does not support DFU yet.
+CONFIG_DESKTOP_CONFIG_CHANNEL_DFU_ENABLE=n
+
+################################################################################
+# Zephyr Configuration
+
+# Use code partition defined in the DTS.
+CONFIG_USE_DT_CODE_PARTITION=y
+
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=1536
+CONFIG_ISR_STACK_SIZE=1536
+CONFIG_MAIN_STACK_SIZE=840
+CONFIG_BT_RX_STACK_SIZE=2048
+CONFIG_BT_HCI_TX_STACK_SIZE_WITH_PROMPT=y
+CONFIG_BT_HCI_TX_STACK_SIZE=1536
+
+CONFIG_BOOT_BANNER=n
+
+CONFIG_NUM_COOP_PRIORITIES=10
+CONFIG_NUM_PREEMPT_PRIORITIES=11
+
+CONFIG_HEAP_MEM_POOL_SIZE=4096
+
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
+CONFIG_SYSTEM_CLOCK_NO_WAIT=y
+
+CONFIG_HW_STACK_PROTECTION=y
+CONFIG_RESET_ON_FATAL_ERROR=n
+
+CONFIG_GPIO=y
+CONFIG_REBOOT=y
+
+CONFIG_SPEED_OPTIMIZATIONS=y
+
+CONFIG_LED=y
+CONFIG_LED_GPIO=y
+
+CONFIG_BT_MAX_PAIRED=2
+CONFIG_BT_ID_MAX=3
+
+CONFIG_BT_CTLR_SDC_LLPM=y
+CONFIG_BT_CTLR_TX_PWR_DYNAMIC_CONTROL=y
+CONFIG_BT_CONN_TX_MAX=6
+
+################################################################################
+
+CONFIG_ASSERT=y
+CONFIG_ASSERT_LEVEL=2
+
+# nRF54L PDK uses UART for logs.
+CONFIG_SERIAL=y
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+CONFIG_LOG=y
+CONFIG_LOG_BACKEND_UART=y
+CONFIG_LOG_BACKEND_SHOW_COLOR=n
```

Comment:
isn't it by default "n"?

Reply:
nRF Desktop's logging configuration updates default of the `CONFIG_LOG_BACKEND_SHOW_COLOR` if `CONFIG_LOG_BACKEND_RTT` is enabled. I would set this Kconfig option here explicitly for consistency (and to avoid relying on default).